frontend/src/Components/UserProfileInvites.tsx

```diff
@@ -63,6 +64,8 @@ export const UserProfileInvites = observer((props: UserProfileInvitesProps) => {
             const result = await api.inviteAPI.create(reason);
             console.log('CREATE', result);
             forceRefresh();
+            props.onInvitesChange();
+            // eslint-disable-next-line
```

Comment:
Why are these necessary?

Reply:
you cannot type exception callback argument (typescript limitation) but lint is complaining about it